packages/terra-slide-panel/src/_variables.scss

```diff
@@ -1,4 +1,8 @@
-$terra-slide-panel-panel-width-small: 320px !default;
-$terra-slide-panel-panel-width-large: 60% !default;
-$terra-slide-panel-panel-border-color: $terra-grey-20 !default;
-$terra-slide-panel-shadow-color: $terra-grey-30 !default;
+// Variables
+// ==========================
+
+$terra-bidi: true;
```

Comment:
I just took a peak, I don't think we need to add `$terra-bidi` for the SlidePanel.scss file.